### ISSUE TITLE
Point.pixels() different results based on start level.

### ISSUE THREAD

**Issue by telemmaite** [geometalab/pyGeoTile#17]
Let`s say we take a coordinate and we try to find it`s parent 4 levels upper.
So we have coord `200,180` and we would like to know it\`s parent 4 levels up which should be `13,12` if we start at 1 or `12,11` if we start at 0.

level|from pix|to pix
-- | -- | --
1 | 0 | 15
2 | 16 | 31
3 | 32 | 47
4 | 48 | 63
5 | 64 | 79
6 | 80 | 95
7 | 96 | 111
8 | 112 | 127
9 | 128 | 143
10 | 144 | 159
11 | 160 | 175
12 | 176 | 191
13 | 192 | 207
14 | 208 | 223
15 | 224 | 239
16 | 240 | 255

However with the code below, we get different results based on our starting zoom level.
We get (`12,11` vs `13,11`)

```python
from pygeotile.point import Point
from pygeotile.tile import Tile

zoom1 = 4
zoom2 = 8
# 8-4 = 4 (delta)


t1 = Point.from_pixel(200, 180, zoom1)
t2 = Point.from_pixel(200, 180, zoom2)


print(t1.pixels(zoom1),t2.pixels(zoom1))
#(200, 180) (13, 11)

zoom1 = 1
zoom2 = 5
# 5-1 = 4 (delta)

t1 = Point.from_pixel(200, 180, zoom1)
t2 = Point.from_pixel(200, 180, zoom2)


print(t1.pixels(zoom1),t2.pixels(zoom1))
#(200, 180) (12, 11)
```

**Comment by Murthy10** [comment]
Pixel values on different zoom levels result in different tiles.

For a better illustration have a look at: https://developers.google.com/maps/documentation/javascript/examples/map-coordinates

But maybe I miss the point of your issue. 
Is there an error in the calculation of pyGeoTile?

**Comment by Murthy10** [comment]
Due to no further corespondents this closes #17

**Comment by Murthy10** [comment]
This closes #17

**Comment by telemmaite** [comment]
Sorry for my very late response.

Yes I do believe there is an issue somewhere. And my issue was solved by using `mercantile.tile` to convert my latitude and longitudes. Here is where I noticed the difference visible in one pixel but throwing out of bounds for my 256 x 256 tiles that I was generating.
level is set to 5, which maps to level 5+8 = 13 for mercantile.
All of the longitude, latitude in the log below give a difference of 1 pixel between both.
Initially I was thinking all difference is because drawing 1->256 compared to 0->255 but, with pygeotile I have img[0,256] which is effectively 257pixels.

```python

point = Point.from_latitude_longitude(latitude = latitude, longitude = longitude)
pixels2 = point.pixels(zoom=level)

pixels = mercantile.tile(longitude, latitude, level+8)

if pixels2[0] != pixels.x or pixels2[1] != pixels.y:
    print(longitude, latitude, pixels, pixels2)

```

```txt
-19.2117 28.9743 Tile(x=3658, y=3406, z=13) (3659, 3407)
-19.2361 28.9316 Tile(x=3658, y=3407, z=13) (3658, 3408)
-19.1343 29.0724 Tile(x=3660, y=3404, z=13) (3661, 3404)
-19.0179 29.0514 Tile(x=3663, y=3404, z=13) (3663, 3405)
-19.0324 29.0121 Tile(x=3662, y=3405, z=13) (3663, 3406)
-19.076 28.9983 Tile(x=3661, y=3406, z=13) (3662, 3406)
-19.1279 28.9451 Tile(x=3660, y=3407, z=13) (3661, 3407)
-19.0403 28.9908 Tile(x=3662, y=3406, z=13) (3663, 3406)
-18.9975 28.986 Tile(x=3663, y=3406, z=13) (3664, 3406)
-18.9963 28.935 Tile(x=3663, y=3407, z=13) (3664, 3408)
-18.9535 29.4849 Tile(x=3664, y=3393, z=13) (3665, 3393)
-18.8229 29.5255 Tile(x=3667, y=3392, z=13) (3668, 3392)
-18.8555 29.481 Tile(x=3666, y=3393, z=13) (3667, 3393)
-18.9591 29.4273 Tile(x=3664, y=3394, z=13) (3665, 3395)
-18.9285 29.422 Tile(x=3665, y=3394, z=13) (3665, 3395)
-18.8984 29.4166 Tile(x=3665, y=3395, z=13) (3666, 3395)
-18.8641 29.458 Tile(x=3666, y=3394, z=13) (3667, 3394)
-18.8462 29.4375 Tile(x=3667, y=3394, z=13) (3667, 3395)
-18.8533 29.41 Tile(x=3666, y=3395, z=13) (3667, 3395)
-18.7825 29.5161 Tile(x=3668, y=3392, z=13) (3669, 3393)
-18.6453 29.4791 Tile(x=3671, y=3393, z=13) (3672, 3393)
-18.785 29.4456 Tile(x=3668, y=3394, z=13) (3669, 3394)
-18.7449 29.3913 Tile(x=3669, y=3395, z=13) (3669, 3396)
-18.6817 29.4302 Tile(x=3670, y=3394, z=13) (3671, 3395)
-18.6377 29.453 Tile(x=3671, y=3394, z=13) (3672, 3394)
-18.6916 29.3827 Tile(x=3670, y=3395, z=13) (3671, 3396)
-18.6569 29.3858 Tile(x=3671, y=3395, z=13) (3671, 3396)
-18.903 29.3572 Tile(x=3665, y=3396, z=13) (3666, 3397)
-18.9328 29.3171 Tile(x=3665, y=3397, z=13) (3665, 3398)
-18.8611 29.38 Tile(x=3666, y=3396, z=13) (3667, 3396)
-18.8128 29.3446 Tile(x=3667, y=3396, z=13) (3668, 3397)
-18.8174 29.3192 Tile(x=3667, y=3397, z=13) (3668, 3398)
-18.9411 29.2569 Tile(x=3664, y=3399, z=13) (3665, 3399)
-18.8985 29.2392 Tile(x=3665, y=3399, z=13) (3666, 3400)
```
